docker/install/ubuntu_install_qemu.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+set -u
+set -o pipefail
+
+apt-get -y install qemu-system
```

Comment:
Should we build a newer qemu from source so that we can have newer architectural features rather than the one packaged by the distros ? My thinking is this could get us something newer.

Happy for a follow up rather than it being in this patch .

Ramana

Reply:
that's a great point, I didn't realize how far behind 18.04 is. will definitely consider it as a follow-up (or if others want to try this, that would also be fantastic!).

Reply:
I wound up upgrading qemu to rule out some problems when I was debugging the integration. updated to 5.1.0 in the latest patch